roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -15,22 +15,37 @@ class DockerImageAvailability(OpenShiftCheck):
 
     skopeo_image = "openshift/openshift-ansible"
 
-    # FIXME(juanvallejo): we should consider other possible values of
-    # `deployment_type` (the key here). See
-    # https://github.com/openshift/openshift-ansible/blob/8e26f8c/roles/openshift_repos/vars/main.yml#L7
     docker_image_base = {
         "origin": {
             "repo": "openshift",
             "image": "origin",
         },
+        "online": {
```

Comment:
After talking with @sdodson, he linked me to https://github.com/openshift/openshift-ansible/blob/8e26f8cf9594693702fd1e545b47116cbbff34f0/roles/openshift_facts/library/openshift_facts.py#L1741-L1771 which is where required images for each deployment type are defaulted. He recommended not worrying about `atomic-enterprise`, and it looks like the cases for `origin` and `online` look to be the same